Source/WebGPU/WebGPU.xcodeproj/project.pbxproj

```diff
@@ -617,6 +640,14 @@
 			path = AST;
 			sourceTree = "<group>";
 		};
+		97296763299BFB72001C8BD4 /* generator */ = {
+			isa = PBXGroup;
+			children = (
+				97296767299D33B9001C8BD4 /* main.js */,
+			);
+			path = generator;
+			sourceTree = "<group>";
+		};
```

Comment:
Instead of running the generator on every build, would it suffice just to update the generated files when we need to change `TypeDeclarations.js`? I don't see the benefit of having everyone run this step all the time when once the WGSL compiler is stable this file won't change very often and isn't much burden for WebGPU devs to update the generated code and submit it.

Reply:
Also given the issues of running `jsc` in the build causing the failures on EWS.

Reply:
I'd think it's more convenient to generate at build time, given it only takes <20ms. I'm chatting with folks to try and figure out the right way to invoke JSC at build time. I think adding a manual step is a bit error prone, and I don't love the idea of checking in generated files, but I'm not fundamentally opposed.

Reply:
I agree with Tadeu. Let's not check in generated files, if we can help it.